run.sh

```diff
@@ -0,0 +1 @@
+nix-shell --run 'cabal new-run exe:realworld-conduit -- -d "postgres:///conduit" -s "Ug3DSsjIdC6ptLtVBoyqdKaIhi90GOHJ11srNN27FezBWAskC5NPbbFiCYW4UwsLfzuwrb4VmOGIRUMCcghFzVi4X6dqkR46e4fXVWpAsILFmBKVmVf4qiTNnQMzM3Nwv3cd44pfPcuNyerH5mfcJR8Yibn2NGyn0YSXtbzcSmS2jjMKZkSszaXe1yyb0O5U4b39ko4G4PqASOoHweAUxl5QEZLMuK5wCaGvi3xKsauub8hrOOTUpjWHhDO9vChg"'
```

Comment:
This is a nit: there are a few dev scripts in the `dev` dir, this could go in there? Ah, and you've reminded me I need to add a big ol' unsafe JWT key to the `.envrc.example`, this one'll do :P